doc/languages-frameworks/index.xml

```diff
@@ -9,7 +9,7 @@
  <xi:include href="android.section.xml" />
  <xi:include href="beam.section.xml" />
  <xi:include href="bower.xml" />
- <xi:include href="coq.xml" />
+ <xi:include href="coq.section.xml" />
```

Comment:
Shouldn't this has a md extension?

Reply:
Currently the markdown gets converted back into docbook first and that is what gets rendered to html. This reads that markdown -> docbook converted output.